arm-resources/links/2016-09-01/swagger/links.json

```diff
@@ -0,0 +1,309 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ManagementLinkClient",
+    "version": "2016-09-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{linkId}": {
+      "delete": {
+        "tags": [
+          "ResourceLinks"
+        ],
+        "operationId": "ResourceLinks_Delete",
+        "description": "Deletes a resource link.",
+        "parameters": [
+          {
+            "name": "linkId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The fully qualified Id of the resource link. For example, /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myGroup/Microsoft.Web/sites/mySite/Microsoft.Resources/links/myLink",
```

Comment:
Still works, according to my tests: https://github.com/Azure/azure-sdk-for-net/pull/2462/files#diff-9b79bc6941ca9f099a07d8fdd87f70c2R235